isis/tests/Fixtures.cpp

```diff
@@ -253,7 +255,110 @@ namespace Isis {
     delete threeImageOverlapFile;
     delete twoImageOverlapFile;
   }
-  
+
+  void ApolloNetwork::SetUp() {
+    TempTestingFiles::SetUp();
+    
+    isdFile1 = new FileName("data/apolloNetwork/Sub4-AS15-M-0583_msk.isd");
```

Comment:
usually names are generic or used to define the kind of images they represent (e.g. steropairL/steropairR or mroLabel.pvl or image1, image2, image3  if it doesn't matter). Could these simply be "apolloImage<n>.pvl/.isd? or something similar?

Reply:
Yeah, they can just be apolloImage1 -> n.